Test/git-issues/git-issue-401.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %dafny /compile:0  "%s" > "%t"
+// RUN: %dafny /compile:0 /z3exe:"%S"/../../Binaries/z3/bin/z3 "%s" >> "%t"
+// RUN: cp -r "%S"/../../Binaries/z3/bin "%S"/../../Binaries/z3/binx
+// RUN: %dafny /compile:0 /z3exe:"%S"/../../Binaries/z3/binx/z3 "%s" >> "%t"
+// RUN: rm -rf "%S"/../../Binaries/z3/binx
```

Comment:
Rather than mess with creating more stuff in the `Binaries` directory, I suggest copying to (and then deleting) `Output/binx`.